plugins/org.jkiss.dbeaver.erd.ui/src/org/jkiss/dbeaver/erd/ui/notations/ERDAssociationType.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * DBeaver - Universal Database Manager
+ * Copyright (C) 2010-2023 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jkiss.dbeaver.erd.ui.notations;
+
+public enum ERDAssociationType {
+    ZERO,
+    ZERO_OR_ONE,
+    ONE_ONLY,
```

Comment:
As we already have `ZERO`, let it be called similarly without the `ONLY` prefix
```suggestion
    ONE,
```

Reply:
Don't think so, prefer to keep all notations defined in specification, ONE and ONLY_ONE is different take a look on screens:
![Screenshot 2023-09-20 085534](https://github.com/dbeaver/dbeaver/assets/32031248/825a66a4-ff3d-42c4-a178-2258a34379eb)

Reply:
Yeah, we might want to add ONE later. Currently we can't check uniqueness to build such associations, however, we have an old issue about supporting/checking/showing unique constraints in erd.